entrypoint.sh

```diff
@@ -320,23 +334,23 @@ echo "$INPUT_IDE_VERSIONS" | while read -r IDE_VERSION; do
   ZIP_FILE_PATH="$HOME/$IDE-$VERSION.zip"
 
   echo "Downloading $IDE $IDE_VERSION from [$DOWNLOAD_URL] into [$ZIP_FILE_PATH]..."
-  CURL_RESP=$(curl -L --silent --show-error -w 'HTTP/%{http_code} - content-type: %{content_type}' --output "$ZIP_FILE_PATH" "$DOWNLOAD_URL")
+  
+  CURL_RESP=$(curl -L --silent --show-error -w '%{json}' --output "$ZIP_FILE_PATH" "$DOWNLOAD_URL")
+  http_code==$(echo "${CURL_RESP}" | jq -r '.response_code // empty')
+  content_type==$(echo "${CURL_RESP}" | jq -r '.content_type // empty')
 
   gh_debug "Checking response code and content type for the download of [$DOWNLOAD_URL] to ensure download successful..."
-  # Turn off 'exit on error'; if we error out when testing the response code,
-  # we want to first print a friendly message to the user and then exit.
-  set +o errexit
-  echo "$CURL_RESP" | grep "HTTP/200 - content-type: application/octet-stream"
-  if [[ $? -eq 0 ]]; then
+
+  if [[ ${http_status} --eq 200 ]] && is_valid_platform_response_content_type "${content_type}"; then
```

Comment:
```suggestion
  if [ "$http_status" -eq 200 ] && is_valid_platform_response_content_type "${content_type}"; then
```

this seems to work for me:

<img width="1152" alt="Screenshot 2023-11-16 at 07 44 05" src="https://github.com/ChrisCarini/intellij-platform-plugin-verifier-action/assets/6374067/4e4e8b54-4a31-4324-b472-d6e58a0add86">


fwiw, works under both versions of bash I tested against:
```
$ > /opt/homebrew/bin/bash test.sh 
Bash version: 5.2.15(1)-release
GOOD

$ > /bin/bash test.sh 
Bash version: 3.2.57(1)-release
GOOD
```

@le-yams - mind giving this a try just to confirm it works on the GitHub Actions runners?